spec/modulr/api/accounts_service_spec.rb

```diff
@@ -41,4 +37,25 @@
       expect(created_account.available_balance).to be_nil
     end
   end
+
+  describe "accounts close" do
+    before do
+      stub_request(:post, %r{/accounts/A121AHGZ/close}).to_return(
+        read_http_response_fixture("accounts/close", "success")
+      )
+    end
+
+    let!(:method_response) do
+      accounts.close(account_id: "A121AHGZ")
+    end
+
+    it_behaves_like "builds correct request", {
+      method: :post,
+      path: "https://api-sandbox.modulrfinance.com/api-sandbox-token/accounts/A121AHGZ/close",
```

Comment:
what do you think is better to use @Nachooc5 ?

1. path: "https://api-sandbox.modulrfinance.com/api-sandbox-token/accounts/A121AHGZ/close",

2. %r{/accounts/A121AHGZ/close}

🤔 
I do not care just to be consistent and in order to not repeat host name I would prefer 2 but I consider more accurate option 1.